lib/sensor.js

```diff
@@ -223,6 +231,20 @@ function Sensor(opts) {
         return this.constrained;
       }
     },
+    freq: {
+      get: function() {
+        return priv.get(this).freq;
+      },
+      set: function(newFreq) {
+        priv.set(this, {
+          freq: newFreq
+        });
```

Comment:
> Since the existing unittests did not notice this, I also need to figure out what should be there to catch that.

After setting `sensor.freq` the values returned by any other getter would've been undefined—that might be useful for ensuring integrity of private data